### ISSUE TITLE
Docs for "experimental" features

### ISSUE THREAD

**Issue by SchrodingersGat** [inventree/InvenTree#5022]
### Body of the issue

Ref: https://github.com/inventree/InvenTree/pull/4982

- In the "getting started" space, we need a page discussing how we handle experimental features
- How to enable / disable features
- A list of currently available features
- Information on the "lifecycle" of such features

For each current experimental feature in the codebase, a page describing the feature in particular

**Comment by matmair** [comment]
I think a paragraph should be enough per feature - the docs are getting hard to navigate as they are.

**Comment by matmair** [comment]
@SchrodingersGat I do not think we will ship anything experimental for 0.12.0 - so do we need to include the docs with 0.12.0 or can we move this to 0.13.0?

**Comment by SchrodingersGat** [comment]
@matmair moved to 0.13.0 :)

**Comment by SchrodingersGat** [comment]
@matmair this one has languished for a while - we have never implemented any "experimental" feautres. I would vote that we remove this entirely to simplify the codebase. Are you using this feature?

**Comment by matmair** [comment]
I am using it in some of my plugins to control features - maybe #8638 is a good candidate

**Comment by matmair** [comment]
Is this enough: https://docs.inventree.org/en/latest/settings/experimental/ or what specifically do you want to have added?

**Comment by SchrodingersGat** [comment]
@matmair I think that is sufficient, thanks!
